hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/RegionReplicaReplicationEndpoint.java

```diff
@@ -631,7 +633,10 @@ public ReplicateWALEntryResponse call(HBaseRpcController controller) throws Exce
             ReplicationProtbufUtil.buildReplicateWALEntryRequest(entriesArray, location
                 .getRegionInfo().getEncodedNameAsBytes(), null, null, null);
         controller.setCellScanner(p.getSecond());
-        return stub.replay(controller, p.getFirst());
+        ReplicateWALEntryResponse response = stub.replay(controller, p.getFirst());
+        long ageLastAppliedOp = entriesArray[entriesArray.length -1].getKey().getWriteTime();
```

Comment:
Because edits are applied in order, we can simply pull the last one, similar to how it's already handled for inter-cluster replication https://github.com/HubSpot/hbase/blob/078e2619ea01889e19b4419ac3eab66c4ac1d3ae/hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSink.java#L276